ods_ci/tasks/Resources/RHODS_OLM/install/oc_install.robot

```diff
@@ -264,7 +264,7 @@ Wait For DSCInitialization CustomResource To Be Ready
 Apply DataScienceCluster CustomResource
     [Documentation]
     [Arguments]        ${dsc_name}=${DSC_NAME}
-    ${file_path} =    Set Variable    tasks/Resources/Files/
+    ${file_path} =    Set Variable    ods_ci/tasks/Resources/Files/
```

Comment:
this is going to break CI/CD

Reply:
it's strange because on local, that part of the path was necessary, but on ci/cd is it different?

Reply:
changed to work on CI/CD

Reply:
hmm i don't think `./ods_ci/..` is  ok for CI/CI. you should leave as it was